keps/prod-readiness/sig-node/2411.yaml

```diff
@@ -0,0 +1,5 @@
+kep-number: 2411
+stable:
+  approvers:
```

Comment:
s/approvers/approver/

@deads2k is taking it

Reply:
Done, thank you!